scripts/data_ingestion/fetch_espn_mlb.py

```
@@ -0,0 +1,657 @@
+
+    # Start run logging
+    script_name = os.path.basename(__file__)
+    command_args = vars(args)
+    run_id = start_run("espn_api", script_name, command_args)
+
+    if not run_id:
+        print("Warning: Failed to start run logging, continuing without tracking")
+
+    try:
+        if args.command == "schedule":
+            date = args.date or datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)